implementation/src/main/java/io/smallrye/config/ConfigMappingInterface.java

```diff
@@ -27,6 +28,22 @@
  * Information about a configuration interface.
  */
 public final class ConfigMappingInterface implements ConfigMappingMetadata {
+
+    private static final Map<Class<?>, Class<?>> PRIMITIVES_TO_WRAPPERS;
+    static {
+        Map<Class<?>, Class<?>> map = new HashMap<>();
+        map.put(boolean.class, Boolean.class);
+        map.put(char.class, Character.class);
+        map.put(byte.class, Byte.class);
+        map.put(short.class, Short.class);
+        map.put(int.class, Integer.class);
+        map.put(long.class, Long.class);
+        map.put(float.class, Float.class);
+        map.put(double.class, Double.class);
+
+        PRIMITIVES_TO_WRAPPERS = Collections.unmodifiableMap(map);
+    }
```

Comment:
Is there perhaps a better place to put this?

Reply:
PR updated. WDYT?

Reply:
Looks great! Thanks!